docs/adr/0028-use-react-testing-library.md

```diff
@@ -0,0 +1,45 @@
+
+# *Use react-testing-library in place of Enzyme*
+
+**User Story:**  *[ES 809](https://jiraent.cms.gov/browse/ES-809)*
+
+  Currently, our frontend test coverage is abysmal. It's sitting right around 30%.
+  As a team, we've talked about being more intentional about writing unit
+  tests, but often times unit tests were taking longer than the feature
+  themselves. We realized that writing tests with [Enzyme](https://enzymejs.github.io/enzyme/)
+  was making testing harder and confusing.
+
+[Enzyme](https://enzymejs.github.io/enzyme/) was released in 2016 by AirBnb.
+At the time, it was the gold standard for testing in the React ecosystem.
+However by 2019, the React API has changed dramatically and support for
+[Enzyme](https://enzymejs.github.io/enzyme/) has decreased with  [react-testing-library](https://testing-library.com/docs/react-testing-library/intro/)
+picking up the market share.
+
+**Sources**
+[Time to say goodbye - Enzyme.js](https://www.piotrstaniow.pl/goodbye-enzyme)
+
+## Considered Alternatives
+
+* [react-testing-library](https://testing-library.com/docs/react-testing-library/intro/)
+  * It has the most support in the industry. This means we will have better
+  resources to help encourage us to write more and better tests.
+  * react-testing-library uses the DOM, which means no more confusing shallow vs.
+  deep rendering.
+  * Accessibility first: RTL provides simple ways to target elements by
+  [role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles)
+  or [accessible name](https://www.w3.org/TR/accname-1.1/).
+  It also has developer friendly ways to test keyboard actions and focus. This was
+  all difficult to do in Enzyme.
+  * Easy to use selectors
+
```

Comment:
I'd add here some "cons" of using RTL: 
- engineers having to learn a new library
- having to either convert existing enzyme tests to RTL or have 2 testing libraries in the code base, which can be confusing.  

I think it's helpful to include those to show that we've taken that into consideration.